### ISSUE TITLE
format of locations information in error output?

### ISSUE THREAD

**Issue by faassen** [dittos/graphqllib#54]
The wsgi_graphql server models itself on the express-graphql server and tries to give the same HTTP output where possible.

I notice that location information looks like this with graphqllib after serializing it to JSON:

```
   "locations": [
           [1, 2]
    ],
```

I expected something like this:

```
'locations': [{'line': 1, 'column': 2}]
```
I think this is because what gets serialized is a Python named tuple that is passed through ``json.dumps`.

Should `format_error` in `graphqllib.core.error` take care of making it the same JSON structure, or should I introduce code in wsgi_graphql that takes care of this?

**Comment by faassen** [comment]
I ask about `format_error` as this is the code that express-graphql uses to format the errors before output.

**Comment by dittos** [comment]
Now `format_error` converts it to a dict. Fixed by 5a5453049eefc47e9de7a520a417be49b8524fad

**Comment by faassen** [comment]
Thanks, this behaves properly now in wsgi_graphql.
